apps/rush-lib/src/cli/actions/CheckAction.ts

```diff
@@ -23,6 +23,7 @@ export class CheckAction extends BaseRushAction {
         "Checks each project's package.json files and ensures that all dependencies are of the " +
         'same version throughout the repository.',
       safeForSimultaneousRushProcesses: true,
+      suppressStartupBanner: true,
```

Comment:
You think this command should never print the startup banner?

Reply:
The startup banner is noise that reduces the clarity of the error. Ideally we'd implement alternative reporters for CI such that we could, e.g. use Azure DevOps logging commands to specify exactly the problem (instead of just exiting with code 1), but that's not really in scope here. Currently `rush check` having the startup banner is a nuisance.